packages/client/src/WalletAPIClient.ts

```diff
@@ -53,12 +53,15 @@ export class WalletAPIClient extends RpcNode<
    */
   public bitcoin: Bitcoin;
 
+  public wallet: Wallet;
```

Comment:
Took another approach and renamed module classes in `somethingModule` but kept the defined instance of such module as `something`.

This in order to keep a streamlined api from the user side, like `client.wallet.method()` instead of `client.walletModule.method()`